pyeudiw/tests/federation/create_tc.py

```diff
@@ -111,58 +53,21 @@
   		      "subset_of": ["RS256", "RS512", "ES256", "ES512", "PS256", "PS512"]
           	}
           	}
-   	 },
-    "trust_marks": [
-   	 {
-   		 "id": "https://trust-anchor.example.eu/openid_relying_party/public/",
-   		 "trust_mark": "eyJhb …"
    	 }
-    ]
-}
-
-# the leaf publishes the leaf public key
-intermediate_es["jwks"]['keys'][0] = leaf_jwk.serialize()
-
+    )
 
-ta_es = {
-	"exp": 1649623546,
-	"iat": 1649450746,
-	"iss": "https://trust-anchor.example.eu",
-	"sub": "https://intermediate.eidas.example.org",
-	"jwks": {
-    	"keys": [
-        	{
-            	"kty": "RSA",
-            	"n": "5s4qi …",
-            	"e": "AQAB",
-            	"kid": "em3cmnZgHIYFsQ090N6B3Op7LAAqj8rghMhxGmJstqg",
-       "x5c": ["..."]
-        	}
-    	]
-	},
-	"trust_marks": [
+trust = TrustedAnchorInfo(
+	10, 
+	"https://trust-anchor.example.eu",
+	"https://intermediate.eidas.example.org",
+	[
     	{
         	"id": "https://trust-anchor.example.eu/federation_entity/that-profile",
         	"trust_mark": "eyJhb …"
     	}
 	]
-}
-
-# the ta publishes the intermediate public key
-ta_es["jwks"]['keys'][0] = intermediate_jwk.serialize()
-
-
-leaf_signer = JWS(leaf_ec, alg="ES256", typ="application/entity-statement+jwt")
-leaf_ec_signed = leaf_signer.sign_compact([leaf_jwk])
-
-intermediate_signer = JWS(intermediate_es, alg="ES256", typ="application/entity-statement+jwt")
-intermediate_es_signed = intermediate_signer.sign_compact([intermediate_jwk])
+)
 
-ta_signer = JWS(ta_es, alg="ES256", typ="application/entity-statement+jwt")
-ta_es_signed = ta_signer.sign_compact([ta_jwk])
+trust_chain = gen_static_trustchain(leaf, intermediate, trust)
 
-trust_chain = [
-	leaf_ec_signed,
-	intermediate_es_signed,
-	ta_es_signed
-]
+print(json.dumps(trust_chain))
```

Comment:
rimuovi il print

Reply:
Resolved in [674b505](https://github.com/italia/eudi-wallet-it-python/pull/74/commits/674b5058b78aecb7a1754295397bd599e004ccea)